aries-site/src/pages/templates/ascending-navigation.mdx

```diff
@@ -0,0 +1,45 @@
+import { DiagramPreview, Example } from '../../layouts';
+
+When an application’s hierarchy is well designed, the ascending button will typically take the user 
+back to the previous page they came from.
+
+<Example 
+  caption="caption TBD" 
+  plain 
+  pad="small"
+  previewWidth="medium"
+  height={{ min: 'small' }}
+>
+  <DiagramPreview 
+    src={{
+      dark: "/templateImages/ascending-button-preview-invert.png",
+      light: '/templateImages/ascending-button-preview.png'
+    }} 
+    alt="caption TBD" />
+</Example>
+
+## Guidance
+- Follows the default button styling with a left arrow icon.
+- Displayed on the left, top corner of the application.
```

Comment:
It's not really just the "left, top" corner. It seems like there is more specific guidance we could provide -->

"Place in the page header above any page content and aligned with the left edge of the content." Or something like that (depending on where we land design-wise).

Reply:
This is still WIP. On Wed. I had a chat w/ Aditi on placement since we've been seeing it be placed in different areas of each service team. We both agreed it should be aligned with content. So this will be updated.

Reply:
Displayed below the header and aligned to the left with the page's content.